src/Serialization/LieTheory.jl

```diff
@@ -75,7 +77,7 @@ end
 type_params(w::WeightLatticeElem) = TypeParams(WeightLatticeElem, parent(w))
 
 function save_object(s::SerializerState, w::WeightLatticeElem)
-  save_object(s, _vec(coefficients(w)))
+  save_object(s, Oscar._vec(coefficients(w)))
```

Comment:
```suggestion
  save_object(s, _vec(coefficients(w)))
```
this gets imported above, so no qualification needed